recipes/civetweb/all/conanfile.py

```diff
@@ -147,9 +142,10 @@ def package(self):
         cmake.install()
         rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
         bin_folder = os.path.join(self.package_folder, "bin")
-        for bin_file in os.listdir(bin_folder):
-            if not bin_file.startswith("civetweb"):
-                os.remove(os.path.join(bin_folder, bin_file))
+        if self.options.with_server_executable:
+            for bin_file in os.listdir(bin_folder):
+                if not bin_file.startswith("civetweb"):
+                    os.remove(os.path.join(bin_folder, bin_file))
```

Comment:
you should restore previous code here

Reply:
@SpaceIm 
Thank you!

Reply:
@SpaceIm 
Do you pointed out `if self.options.with_server_executable` should be reverted?
I think it is necessary modification as I have add `with_server_executable` option.

Reply:
Why would it be not necessary to remove msvc runtime files?